lua/cps.lua

```diff
@@ -5,6 +5,7 @@ return function (net_conf, gk_conf, gt_conf, lls_conf, numa_table)
 	--
 
 	-- These parameters should likely be initially changed.
+	local kni_kmod_path = "/home/gatekeeper/gatekeeper/dependencies/dpdk/build/kmod/rte_kni.ko"
```

Comment:
You can drop this change from this patch since it has already been addressed in pull request #231.